src/unix.rs

```diff
@@ -232,14 +233,14 @@ fn getpwuid(real: bool) -> OsString {
         fancy_fallback_os(result)
```

Comment:
I'm thinking fancy fallbacks can move to lib.rs, and expose the error in the fallible API